st2actions/st2actions/runners/actionchainrunner.py

```diff
@@ -63,6 +67,35 @@ def init_vars(self, action_parameters):
             self.vars = self._get_rendered_vars(self.actionchain.vars,
                                                 action_parameters=action_parameters)
 
+    def validate(self):
```

Comment:
Yes. Early validation is good for the all the meta stuff. In workflow cases we only can really validate at runtime since the workflow could be changed without StackStorm even knowing. I do that myself all the time.

Therefore, we necessarily have to perform validations at runtime but can additionally do so at create/update time - these might have limited value IMO.